pkg/test/framework/features/whitelist.txt

```diff
@@ -509,6 +509,40 @@ mtls_k8s_ca,TestMtlsStrictK8sCA/global-plaintext/naked->ws://b:http
 mtls_k8s_ca,TestMtlsStrictK8sCA/global-plaintext/naked->ws://headless:http
 mtls_k8s_ca,TestMtlsStrictK8sCA/global-plaintext/naked->ws://multiversion:http
 mtls_k8s_ca,TestMtlsStrictK8sCA/global-plaintext/naked->ws://naked:http
+multicluster_centralistio,TestClusterLocalService
```

Comment:
I think the intent of this was you add the feature labels instead of adding the tests to the whitelist. Ideally its easier to add the feature label @therealmitchconnors

Reply:
@howardjohn so all the tests in test/integration/multicluster/ use the ReachabilityTest:
https://github.com/istio/istio/blob/master/tests/integration/multicluster/reachability.go#L33
example here:
https://github.com/istio/istio/blob/master/tests/integration/multicluster/multimaster/main_test.go#L62
if I am understanding the features requirement, framework.NewTest(t).Feature("my.feature.string") needs to be added to ReachabilityTest - but that test needs to be able to know which test (multimaster / remote / centralistio) is calling it in order to set the feature Tag properly

The Reachability test needs to be modified to handle the features requirement - so I feel that can be addressed in a followup PR